pages/homePage.ts

```diff
@@ -0,0 +1,45 @@
+import { Locator } from "@playwright/test";
+import { URLS } from "../config/urlProvider";
+import { BasePage } from "./basePage";
+
+export class HomePage extends BasePage {
+  readonly addUserLink = this.page.locator(
+    `xpath=//a[@href="${URLS.ADD_USER}"]`,
+  );
+
+  readonly usersTable = this.page.locator(
+    'xpath=//table[@data-testid="table-Users"]',
+  );
+
+  public createdUser: Locator;
```

Comment:
it's a bad approach, you need to use getUserByUserName before using this variable in code and it could be so hard to manage. It is better remove this variable and use for all your needs getUserByUserName method